lib/simple-db.js

```diff
@@ -11,6 +12,12 @@ class SimpleDb {
       return JSON.parse(cats.toString());
     });
   }
+
+  save(obj) {
+    const id = crypto.randomBytes(8).toString('hex');
+    const cats = { ...obj, id };
```

Comment:
an alternative way to write 18 could be:
`obj.id = id`
and replacing `cats` on line 19 with `obj`